api.planx.uk/modules/webhooks/service/sendNotification/index.ts

```diff
@@ -36,7 +36,11 @@ export const sendSlackNotification = async (
   const pilotServices = [
     "uniform",
     "happ",
+    "hapr",
+    "hret",
```

Comment:
`HRET` is what actually came through Slack, but `HAPR` also looks like a valid application type "suffix" in BOPS codebase so cautiously accounting for both! 
- https://github.com/unboxed/bops/blob/main/spec/factories/application_type.rb
- https://github.com/unboxed/bops/blob/main/db/migrate/20240321155240_add_householder_retrospective_type.rb#L14

Reply:
🕵️‍♀️